src/components/ProjectSettingsTabs/AccessTab/getFormSchema.ts

```diff
@@ -79,6 +79,7 @@ export const getFormSchema = (
       admins: {
         "ui:addButtonText": "Add Username",
         "ui:description": getAdminsDescription(projectType),
+        "ui:orderable": false,
```

Comment:
I just implemented a way of ordering items in a list via arrow buttons -- RJSF defaults this setting to true, but most of our pages don't use it, so I had to set them all to false.

Reply:
This might be a silly question but is there a reason we couldn't do something like `const isOrderable = uiSchema["ui:orderable"] ?? false` and rely on that for rendering?